### PR TITLE
Create 2019-08-07-all-hands-meeting.md

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

File: 2019-08-07-all-hands-meeting.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+### Announcements
+
+- Saturday noon until whenever BBQ at Ben's (add to calendar + ping uv), pc + el + dc
+    - this is a post-burnout therapy session, no talking biz
+
+### Working Group Updates
+
+- [Business Planning][biz-wg]
+    - dc: we really need to meet, operating cost needs to land
+    - dc: gi wants to do branding in august
```
Comment (patcon) [inline 2019-08-07-all-hands-meeting.md]: https://github.com/hyphacoop/organizing/issues/77

Testing issue crosslinking

File: 2019-08-07-all-hands-meeting.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+### Announcements
+
+- Saturday noon until whenever BBQ at Ben's (add to calendar + ping uv), pc + el + dc
+    - this is a post-burnout therapy session, no talking biz
+
+### Working Group Updates
+
+- [Business Planning][biz-wg]
+    - dc: we really need to meet, operating cost needs to land
```
Comment (patcon) [inline 2019-08-07-all-hands-meeting.md]: https://github.com/hyphacoop/organizing/issues/85#issuecomment-520323307

File: 2019-08-07-all-hands-meeting.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    - pc: consider small internal use-case? #todo?
+        - bl: for now maybe for random chats
+    - questions
+        - how do we receive USD? maybe transferwise?
+            - patcon: can we just choose a standard thing we can use for any cross-border
+            - dc: money always hard
+        - dc: relying on deposit of USD to CAD account not great (getting hit with bank conversion)
+        - bl: can add a USD account, and transfer whenever rate is good
+            - bl/dc agree
+        - #todo research employees taking USD as payroll
```
Comment (patcon) [inline 2019-08-07-all-hands-meeting.md]: https://github.com/hyphacoop/organizing/issues/84

File: 2019-08-07-all-hands-meeting.md (hunk 1 above)
Comment (benhylau) [reply 2019-08-07-all-hands-meeting.md]: Do you mean for these to be added to notes? What is the purpose of these comments?

Comment (patcon) [reply 2019-08-07-all-hands-meeting.md]: The issue will have crosslinks to what meetings its discussed in

Comment (patcon) [reply 2019-08-07-all-hands-meeting.md]: Could immediately resolve as part of process. Or we could figure out another way to crosslink (dated labels were discussed, but they feel heavy in github)

Comment (benhylau) [reply 2019-08-07-all-hands-meeting.md]: Ahhh I think this works

Comment (benhylau) [reply 2019-08-07-all-hands-meeting.md]: Ahhh I think this works

Comment (benhylau) [reply 2019-08-07-all-hands-meeting.md]: Ahhh I think this works

Comment (benhylau) [reply 2019-08-07-all-hands-meeting.md]: Ahhh I think this works

**Pull request merged**